apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/RdaFissAuditTrail.java

```diff
@@ -54,8 +54,8 @@ public class RdaFissAuditTrail {
   @Column(name = "badt_curr_date")
```

Comment:
Thanks, @KyRobbins. I'm putting this comment on as a temporary hold–I replicated the validation error via the ansible testing harness locally after switching over to postgresql. I trust this is all cleared up now and I'd like to see that reflected in the local testing.

Reply:
Verified locally. Resolving this comment, approving. Thanks, again.